templates/typescript/samples/sample-assistant/src/index.ts

```diff
@@ -77,12 +77,16 @@ const credentialProvider: SimpleCredentialProvider = new SimpleCredentialProvide
 // Register the skills configuration class.
 const skillsConfig: SkillsConfiguration = new SkillsConfiguration(appsettings.botFrameworkSkills as IEnhancedBotFrameworkSkill[], appsettings.skillHostEndpoint);
 
+let authenticationConfiguration = new AuthenticationConfiguration();
+
 // Register AuthConfiguration to enable custom claim validation.
-const allowedCallers: string[] = [...skillsConfig.skills.values()].map(skill => skill.appId);
-const authenticationConfiguration = new AuthenticationConfiguration(
-    undefined,
-    allowedCallersClaimsValidator(allowedCallers)
-);
+if(skillsConfig.skills.size > 0){
```

Comment:
```suggestion
if (skillsConfig.skills.size > 0) {
```

Be careful with the linter. Execute `npm run lint-fix` to fix the automatic fixes